sympy/series/tests/test_limits.py

```diff
@@ -456,6 +456,11 @@ def test_issue_8730():
     assert limit(subfactorial(x), x, oo) == oo
 
 
+def test_issue_10801():
+    # make sure limits work with binomial
+    assert limit(16**k/(k*binomial(2*k,k)**2), k, oo) == pi
```

Comment:
Code style fix noted.
The suggestion for using `gamma._eval_rewrite_as_tractable` is essentially 
```
In [1]: e = 16**k / (k * binomial(2*k, k)**2)

In [2]: limit(e.rewrite(gamma).rewrite('tractable'), k, oo)
Out[2]: π
```
and it works. 
This alone also works:
```
In [3]: limit(e.rewrite(gamma), k, oo)
Out[3]: π
```
Then why does simply ` _eval_rewrite_as_tractable = _eval_rewrite_as_gamma` not work?